src/Proto.Persistence/Persistence.cs

```diff
@@ -28,7 +28,7 @@ public void Init(IProvider provider, IContext context)
             if (t != null)
             {
                 EventIndex = t.Item2;
-                Context.ReceiveAsync(t.Item1).Wait();
+                await Context.ReceiveAsync(t.Item1);

```

Comment:
The method is a void method so that would not compile.
Are you missing any additional changes in the commit?